extensions/wikia/Search/classes/Test/HooksTest.php

```diff
@@ -77,6 +77,11 @@ public function testOnArticleDeleteComplete() {
 		$mockUser = $this->getMockBuilder( 'User' )
 		                 ->disableOriginalConstructor()
 		                 ->getMock();
+		$mockTitle = $this->getMockBuilder( 'Title' )
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$mockTitle" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)